multiqc/modules/qualimap/QM_BamQC.py

```diff
@@ -622,12 +624,20 @@ def general_stats_headers(self):
         "hidden": True,
     }
     self.general_stats_headers["regions_mapped_reads"] = {
-        "title": "{} Aligned".format(config.read_count_prefix),
+        "title": "{} On target".format(config.read_count_prefix),
         "description": "Number of mapped reads on target region ({})".format(config.read_count_desc),
         "scale": "RdYlGn",
         "shared_key": "read_count",
         "hidden": True,
     }
+    self.general_stats_headers["percentage_aligned_on_target"] = {
```

Comment:
I think it makes sense to have this to the left of the more generic columns like "M Aligned", as it's a subset. So better to move further up the code basically.